src/app/components/ParaTimePicker/LayerMenu.tsx

```diff
@@ -34,7 +34,7 @@ export const DisabledLayerMenuItem: FC<BaseLayerMenuItemProps> = ({ divider, lay
   const labels = getLayerLabels(t)
 
   return (
-    <Tooltip arrow placement="top" title={'Coming soon'}>
+    <Tooltip arrow placement="top" title="Coming soon">
```

Comment:
Missing translation

Reply:
Fixed in https://github.com/oasisprotocol/explorer/pull/630